### ISSUE TITLE
Feature request: restrict the date from a dynamic field or datepicker

### ISSUE THREAD

**Issue by UniErfurtURMZ** [znuny/znuny-feature-requests#34]
hi,

we want to restrict the date from a dynamic field or datepicker for our customer, so that you can not select today's date. Currently we use the "default date difference" but the date can still be changed by the customer. 

The idea is that we need to set the date at least one day in the future. For example we need at least one day for printing a poster or at least 3 weeks for setting up new user accounts.

Basically we need the same functions like the JQuery UI Date Picker to restict the date for the customer.
The restrictions must be effective both in the agent frontend and in the customer frontend.
